package.json

```diff
@@ -1,6 +1,12 @@
 {
   "name": "@manifoldco/shadowcat",
   "description": "Invisible web component auth implementation of Manifold's PUMA",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/manifoldco/ui.git"
+  },
+  "author": "manifoldco",
+  "license": "BSD",
```

Comment:
License was incorrect, and missing `repository` field was raising install warnings in UI